Sources/NibLoadable.swift

```diff
@@ -12,9 +12,9 @@ public protocol NibLoadable: class {
 
 public extension NibLoadable {
   /// By default, use the nib which have the same name as the name of the class,
-  /// and located in the main bundle
+  /// and located in the bundle of that class
   static var nib: UINib {
-    return UINib(nibName: String(Self), bundle: nil)
+    return UINib(nibName: String(Self), bundle: NSBundle(forClass: self))
```

Comment:
Just wondering if we should use `Self` or `self` here… (see also #2) open question here.

Might as well use `self` everywhere to solve #2 too in the process